heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/DisabledRateLimitedCache.java

```diff
@@ -24,13 +24,22 @@
 import java.util.concurrent.ConcurrentMap;
 import lombok.RequiredArgsConstructor;
 
+/**
+ * A cache to reduce writes to Elasticsearch.
+ *
+ * Main difference from {@link DefaultRateLimitedCache} is there is no rate limiting enforced.
+ */
 @RequiredArgsConstructor
 public class DisabledRateLimitedCache<K> implements RateLimitedCache<K> {
     private final ConcurrentMap<K, Boolean> cache;
 
     @Override
     public boolean acquire(K key, final Runnable cacheHit) {
-        return cache.putIfAbsent(key, true) == null;
+        if (cache.putIfAbsent(key, true) == null) {
+            cacheHit.run();
+            return false;
```

Comment:
The return values here are inversed from what they were before, is that intentional?

Reply:
no, its not 👎 - that would explain the graphs from yesterday. I added a unit test and fixed the conditional.